cmds/init/init.go

```diff
@@ -31,19 +31,29 @@ var (
 	test     = flag.Bool("test", false, "Test mode: don't try to set control tty")
 	debug    = func(string, ...interface{}) {}
 	osInitGo = func() {}
-	cmdList  = []string{
-		"/inito",
-
-		"/bbin/uinit",
-		"/bin/uinit",
-		"/buildbin/uinit",
-
-		"/bin/defaultsh",
-	}
+	cmdList  []string
 	cmdCount int
 	envs     []string
+	root     = os.Getenv("UROOT_ROOT")
 )
 
+func r(n ...string) string {
```

Comment:
Can this function name be more descriptive?

Reply:
but I still like r oh well